storage/innobase/srv/srv0srv.cc

```diff
@@ -1479,6 +1485,29 @@ srv_export_innodb_status(void)
 
 	export_vars.innodb_buffer_pool_pages_dirty = flush_list_len;
 
+	export_vars.innodb_buffer_pool_pages_made_young = stat.n_pages_made_young;
+	export_vars.innodb_buffer_pool_pages_made_not_young = stat.n_pages_not_made_young;
+	export_vars.innodb_buffer_pool_pages_old = 0;
+	for (i = 0; i < srv_buf_pool_instances; i++) {
+		buf_pool_t*	buf_pool = buf_pool_from_array(i);
+		export_vars.innodb_buffer_pool_pages_old
+			+= buf_pool->LRU_old_len;
+	}
+
+	export_vars.innodb_checkpoint_age = (log_sys.lsn - log_sys.last_checkpoint_lsn);
+	export_vars.innodb_checkpoint_max_age = log_sys.max_checkpoint_age;
+	export_vars.innodb_history_list_length = trx_sys.rseg_history_len;
```

Comment:
We are also missing
```c++
	export_vars.innodb_deadlocks = srv_stats.lock_deadlock_count;
```
We do have `MONITOR_INC(MONITOR_DEADLOCK)` for exactly this, but I think that it would be better to unconditionally increment an atomic counter and then return that from `srv_mon_process_existing_counter()`, like it was done in c83f83705309e34e5d5f35b34be79f2143af2e61. (Note: I want to remove the `MONITOR` interface at some point.)